doc/README.md

```diff
@@ -514,15 +516,32 @@ Expected outcome
 * ARC responds with status `ANNOUNCED_TO_NETWORK` or `SEEN_ON_NETWORK`
 * At latest a couple of minutes later the status will switch to `SEEN_ON_NETWORK`
 
-### Block reorg
+### Chain reorg
 
-A block reorg happens and two different blocks get announced at the same block height
+A chain reorganization (chain reorg) occurs when the blockchain switches its primary chain of blocks, usually because a previously stale chain (fork) has grown to surpass the current longest chain. This process can affect transaction statuses based on their presence in the involved chains.
 
-Expected outcome:
-* ARC does not update the transaction statuses according to the block in the longest chain.
-* Information and Merkle path of the block received first will be persisted in the transaction record and not overwritten
+#### Transaction in both chains
+This is the most common scenario. If a transaction is included in both the original longest chain and the stale chain (which becomes the longest), the transition to the new longest chain does not disrupt the transaction's confirmation status.
+
+A new `MINED` callback will be send for that transaction with updated block data (block hash, block height, Merkle path).
+
+#### Transaction in the longest chain, but not in the stale chain that becomes longest
```

Comment:
That sounds a bit confusing.
How about

Transaction in the **previously** longest chain, but not in the stale chain that becomes longest

Reply:
Sure, added that